### PR TITLE
[processing] Fix "Extract specific vertices" alg help text (release-3_34 branch)

### PR DESCRIPTION
**Pull request by agiudiceandrea**
## Description

The input vector layer for the "Extract specific vertices" (`native:extractspecificvertices`) processing algorithm can be of any geometry type.

Already fixed in master and 3.36 with dca30bd25360fc67a19f13f5f86cbdbc5300a78f (https://github.com/qgis/QGIS/pull/55996).

@alexbruy, it looks like also the "Extract vertices" algorithm has the same issue in the short help text https://github.com/qgis/QGIS/blob/71f3cc70370d1d190984e4350f5d270dc1c36761/src/analysis/processing/qgsalgorithmextractvertices.cpp#L52 while it accepts any spatial vector layer as input.

### CODE REVIEW

File: src/analysis/processing/qgsalgorithmextractspecificvertices.cpp [hunk 1]
```diff
@@ -49,7 +49,7 @@ QString QgsExtractSpecificVerticesAlgorithm::groupId() const
 
 QString QgsExtractSpecificVerticesAlgorithm::shortHelpString() const
 {
-  return QObject::tr( "This algorithm takes a line or polygon layer and generates a point layer with points "
+  return QObject::tr( "This algorithm takes a vector layer and generates a point layer with points "
                       "representing specific vertices in the input lines or polygons. For instance, this algorithm "
```
Comment (DelazJ) [inline src/analysis/processing/qgsalgorithmextractspecificvertices.cpp:53]:
```suggestion
                      "representing specific vertices in the input features. For instance, this algorithm "
```
should we not include (multi)point here also?

File: src/analysis/processing/qgsalgorithmextractspecificvertices.cpp [hunk 2]
```diff
@@ -49,7 +49,7 @@ QString QgsExtractSpecificVerticesAlgorithm::groupId() const
...
                       "can be used to extract the first or last vertices in the geometry. The attributes associated "
                       "to each point are the same ones associated to the line or polygon that the point belongs to." ) +
```
Comment (DelazJ) [inline src/analysis/processing/qgsalgorithmextractspecificvertices.cpp:55]:
```suggestion
                      "to each point are the same ones associated to the feature that the point belongs to." ) 
```
?

**Review by DelazJ — COMMENTED** [review]
should we still keep mention of lines and polygons (only)?

**Comment by agiudiceandrea** [comment]
> should we still keep mention of lines and polygons (only)

Good catch! It looks like @alexbruy forgot to also change these lines of the short help in https://github.com/qgis/QGIS/pull/55996.

Not sure how to proceed now. Maybe this PR should remain as is (just a backport to 3.34 to something already in master and 3.36) and then open another PR for master branch (to be backported to 3.34 and 3.36) in order to add your suggestions for the "Extract specific vertices" and to also fix the ""Extract vertices" alg in the same way.

File: src/analysis/processing/qgsalgorithmextractspecificvertices.cpp (hunk 1 above)
Comment (agiudiceandrea) [reply src/analysis/processing/qgsalgorithmextractspecificvertices.cpp:53]: Maybe here it is better "geometry".

**Comment by DelazJ** [comment]
> Already fixed in master and 3.36 with https://github.com/qgis/QGIS/commit/dca30bd25360fc67a19f13f5f86cbdbc5300a78f (https://github.com/qgis/QGIS/pull/55996).

Maybe I miss something but this PR is not a backport of #55996; only the help text is modified while, I assume, points are still not (yet) supported in 3.34.
And yes, the changes I suggested apply also to master, so a specific PR would be necessary.

**Comment by agiudiceandrea** [comment]
Superseded by https://github.com/qgis/QGIS/pull/56505.

**Pull request closed without merging**